src/runtime_config/default/actions.js

```diff
@@ -74,6 +74,17 @@ function checkNone(args) {
   }
 }
 
+function checkEntityId(value, field) {
+  checkType(value, "string", field);
+  if (value.length !== 64) {
```

Comment:
Perhaps define `64` as a `const` around this line, along with a comment to explain this is the hex-encoded string of a SHA-256 hash?